src/pages/Environments.js

```diff
@@ -424,9 +424,13 @@ const Environments = () => {
               const multipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
                 disksByProject[googleProject]).length > 1
               const forAppText = !!appType ? ` for ${_.capitalize(appType)}` : ''
+              // Galaxy can have multiple disks per project as long as they are in different workspaces
+              const galaxyDisksOnly = _.filter(disk => disk.labels.saturnApplication?.toLowerCase() === tools.galaxy.appType.toLowerCase() && disk.status !== 'DELETING', filteredDisks)
```

Comment:
If you use `getDiskAppType`, it takes care of the case sensitivity issue for you.

Reply:
thanks for the feedback, will update!